server/test/upload-test-data.js

```diff
@@ -0,0 +1,167 @@
+import fs from "fs";
+
+import MindbodyRequest from "../requests.js";
+import MindbodyAccess from "../api-manager.js";
+import { lookup } from "dns";
+
+const fileName = process.argv.slice(2)[0];
+const NUMWEEKS = 3;
+const WEEK = 7;
+
+fs.readFile("./" + fileName, "utf8", (err, jsonString) => {
+  if (err) {
+    console.log("File read failed:", err);
+    return;
+  }
+  try {
+    const clientsList = JSON.parse(jsonString);
+    var promises = new Array();
+    let products;
+    let classes;
+    let clients;
+
+    //gets authorization and then creates an array of promises that return via the Promise.all()
+    MindbodyAccess.getAuth()
+      .then((value) => {
+        for (let i = 0; i < clientsList.clients.length; i++) {
+          promises[i] = MindbodyAccess.postClient(clientsList.clients[i]);
+        }
+        return Promise.all(promises);
+      })
+      .then((value) => {
+        return MindbodyAccess.post;
```

Comment:
What does this post function/data member do?

Reply:
It seems like a copy/paste error that's not causing any harm, but let's remove it before merging this to dev.